testing/deploy_utils.py

```diff
@@ -97,14 +97,22 @@ def setup_kubeflow_ks_app(dir, namespace, github_token, api_client):
   util.run(["ks", "env", "set", "default", "--namespace=" + namespace_name],
     cwd=app_dir)
 
-  kubeflow_registry = "github.com/kubeflow/kubeflow/tree/master/kubeflow"
+  # IDK what the env looks like lets just dump everything
```

Comment:
This hard coded registry to master was confusing to me, and using the one that's checked out seems to work -- is there a reason for the master registry?

Reply:
Why are you modifying this script?

Reply:
Are you using this script test_deploy.py to deploy the spark operator in your cluster?

Reply:
So I figured that hard coding master was a bad idea and installing Spark as part of the e2e minikube tests make sure it at least can be installed. I don't feel strongly about this since we don't need it for the full workflow tests so I'm happy to revert if this complicates matters.